doc/generate_dtype_tensor_table.py

```diff
@@ -18,17 +18,19 @@
         ('matrix', (False,False)),
         ('tensor3', (False,False,False)),
         ('tensor4', (False,False,False,False)),
-        ('tensor5', (False,False,False,False,False)),]
+        ('tensor5', (False,False,False,False,False)),
+        ('tensor6', (False,) * 6),
+        ('tensor7', (False,) * 7),]
 
-hdr = '============ =========== ==== ============ ==================================='
+hdr = '============ =========== ==== ================ ================================================='
 print(hdr)
-print('Constructor  dtype       ndim shape        broadcastable')
+print('Constructor  dtype       ndim shape            broadcastable')
 print(hdr)
 for letter in letters:
     for shape in shapes:
         suff = ',)' if len(shape[1])==1 else ')'
         s = '(' + ','.join('1' if b else '?' for b in shape[1]) + suff
-        print('%s%-10s  %-10s  %-4s %-11s  %-20s' %(
+        print('%s%-10s  %-10s  %-4s %-15s  %-20s' %(
```

Comment:
I didn't look at the output, but I suport it print something like `(False, ....)`. It it isn't too hard, changing it to `(False, ) * 6` would make this more clear I think. Anyway, just a suggestion.